bundles/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/internal/MqttBindingConstants.java

```diff
@@ -42,4 +41,10 @@
     public static final String DATETIME = "datetime";
     public static final String ROLLERSHUTTER = "rollershutter";
     public static final String TRIGGER = "trigger";
+
+    public static enum COLOR_MODE {
```

Comment:
@J-N-K, @jochen314, should I name it COLOR_MODE or ColorMode? :thinking:

Reply:
Camelcase, and the `static` is not needed